docs/structured-streaming-programming-guide.md

```diff
@@ -817,7 +997,14 @@ Here is a table of all the sinks, and the corresponding settings.
     <td><pre>writeStream<br/>  .format("memory")<br/>  .queryName("table")<br/>  .start()</pre></td>
     <td>No</td>
     <td>Saves the output data as a table, for interactive querying. Table name is the query name.</td>
-  </tr> 
+  </tr>
+  <tr>
+    <td></td>
```

Comment:
empty row to make sure that there is line after the table when rendered. otherwise looks odd.